pirate.h

```diff
@@ -13,6 +13,7 @@
 //#define XSTR(x) STR(x)
 //#define STR(x) #x
 //#pragma message "BP_FIRMWARE_HASH value:" XSTR(BP_FIRMWARE_HASH)
+#define BP_FILENAME_MAX 13

```

Comment:
Since we end up doing a +1 in button_scr.c to make space for the null termination of the string, maybe this should be incremented?

//Support 8.3 file name format, with null termination
#define BP_FILENAME_MAX 8 + 1 + 3 +1